shared/login/signup/username-email-form.render.native.js

```diff
@@ -3,15 +3,15 @@ import Container from '../forms/container'
 import React, {Component} from 'react'
 import type {Props} from './username-email-form.render'
 import {UserCard, Input, Button} from '../../common-adapters'
-import {globalColors} from '../../styles/style-guide'
+import {globalColors, globalMargins} from '../../styles/style-guide'
 
 class Render extends Component<void, Props, void> {
   render () {
     return (
       <Container onBack={this.props.onBack} outerStyle={stylesOuter}>
         <UserCard>
-          <Input autoFocus={true} hintText='Create a username' value={this.props.username} errorText={this.props.usernameErrorText} onChangeText={username => this.props.usernameChange(username)} />
-          <Input style={stylesInput2} hintText='Email address' value={this.props.email} errorText={this.props.emailErrorText} onEnterKeyDown={this.props.onSubmit} onChangeText={email => this.props.emailChange(email)} keyboardType='email-address' />
+          <Input autoFocus={true} hintText='Create a username' floatingLabelText='Create a username' value={this.props.username} errorText={this.props.usernameErrorText} onChangeText={username => this.props.usernameChange(username)} />
```

Comment:
> can you remove hintText if you add floating? i thought so...

Only on desktop, not native.  Added a ticket to make the semantics match: DESKTOP-1773.